Tests/SwiftParserTest/Expressions.swift

```diff
@@ -395,8 +395,8 @@ final class ExpressionTests: XCTestCase {
       \#^AFTER_SLASH^#\(#^AFTER_PAREN^#
       """##,
       diagnostics: [
-        DiagnosticSpec(locationMarker: "AFTER_SLASH", message: #"Expected root and expression after '\' in key path"#),
-        DiagnosticSpec(locationMarker: "AFTER_PAREN", message: "Expected value after '(' in tuple"),
+        DiagnosticSpec(locationMarker: "AFTER_SLASH", message: #"Expected root and expression in key path"#),
```

Comment:
Is there an issue up for key paths in general? `root` isn't required, but it looks like we're calling `parsePostfixExpression` if there's no `.`, that returns missing, and thus we get an expected "root" here where that makes no sense (the type isn't required, really I'd expect "Expected '.' and expression in key path" here).